pkg/bro-backend/config.go

```diff
@@ -0,0 +1,42 @@
+package brobackend
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/threeal/bro/pkg/utils"
+)
+
+var CONFIG_FILENAME = "backend_config.json"
+
+type Config struct {
+	ListenAddr *string `json:"listen_addr"`
+}
```

Comment:
I wonder, can we use `string` instead of `*string`?
So instead of checking if it's `nil` we could just check if it's `""` (empty).